README.md

```diff
@@ -53,3 +53,4 @@ For 2.0.0 version, it supports:
 ----------
 ## License
   [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html) Copyright (C) Apache Software Foundation
+
```

Comment:
extra line, remove that

Reply:
sorry, the space line autoadd by vscode.😞